CHANGELOG.md

```diff
@@ -3,9 +3,14 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
+
+## [2.2.24] - 23.01.2022
```

Comment:
You need to update the links at the bottom as well

Reply:
Done